### ISSUE TITLE
Don't include page fragment in linkcheck

### ISSUE THREAD

**Issue by david-venhoff** [digitalfabrik/integreat-cms#1686]
### Motivation

Currently links to pages which include an invalid url fragment are marked as invalid. However, pages usually still work if the url specifies an invalid fragment.

### Proposed Solution

Treat such urls as valid.

**Comment by timoludwig** [comment]
Presumably, these types are not sufficient:
https://github.com/digitalfabrik/integreat-cms/blob/8343fcd7ccbb6547f732c0d639c38826047fcc64/integreat_cms/core/settings.py#L793-L797
Additionally, we should ignore the type `external`.

**Comment by timoludwig** [comment]
Apart from that, we could think about listing all ignored url types under the "ignored" section instead of just hide them to enable the link replacement feature for them.

**Comment by david-venhoff** [comment]
We decided to add a warning category for soft errors, and these links could belong to that category. This would also keep the link replacement feature working.
So this issue is no longer relevant after #1695
